src/zenml/integrations/gcp/step_operators/vertex_step_operator.py

```diff
@@ -220,19 +209,17 @@ def launch(
         client = aiplatform.gapic.JobServiceClient(
             credentials=credentials, client_options=client_options
         )
-        accelerator_count = (
-            resource_settings.gpu_count or settings.accelerator_count
-        )
+
         custom_job = {
             "display_name": info.run_name,
             "job_spec": {
                 "worker_pool_specs": [
                     {
                         "machine_spec": {
-                            "machine_type": settings.machine_type,
-                            "accelerator_type": settings.accelerator_type,
-                            "accelerator_count": accelerator_count
-                            if settings.accelerator_type
+                            "machine_type": settings.instance_type,
```

Comment:
Just a personal note here (feel free to ignore): I am still a bit unsure about this approach here. For instance, the configuration here is called `machine_type` in the Vertex nomenclature and we are calling it `instance_type` now. It is not too bad but it feels a bit unintuitive for me.

Reply:
I have the same concern actually and also talked to @htahir1, and I think we're going with a unified name for now

Reply:
What would be an alternative?

Reply:
We use the name that the tool (Vertex, Sagemaker, Skypilot, ...) uses. From what I've seen, it's "machine_type" for vertex, "size" for azure (which is just unintuitive) and instance_type for the rest

Reply:
but then you can specify "size" for vertex right? or "machine_type" for azure?

Reply:
No, in this case this would not be a general item on `ResourceSettings` (-> no autocomplete) but would be one of the extras that users could specify (still using `ResourceSettings`, just no explicit attribute)

Reply:
In this case, id go with clear docs and a unified attribute. WDYT @bcdurak ?

Reply:
yes lets do a unified attribute and clear docs